tests/test_data/matsim/HK_network.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
I'm assuming this network file differs from the others because it contains elevation data? If so, maybe the file name can make this clear. Something like `hong_kong_network_with_elevation_values.xml`